make.js

```diff
@@ -809,7 +809,7 @@ CLI.bump = function() {
         taskJson.version.Patch = taskJson.version.Patch + 1;
         taskLocJson.version.Patch = taskLocJson.version.Patch + 1;
 
-        fs.writeFileSync(taskJsonPath, JSON.stringify(taskJson, null, 4));
+        fs.writeFileSync(taskJsonPath, JSON.stringify(taskJson, null, 2));
```

Comment:
[no change needed] I looked this up.  The change is to making the spaces consistent, looks good :)